miner/worker.go

```diff
@@ -184,7 +184,12 @@ func (w *worker) commitNewWork() (*types.Block, error) {
 		return nil, fmt.Errorf("failed to create new current environment: %w", err)
 	}
 	// Configure any stateful precompiles that should go into effect during this block.
-	w.chainConfig.CheckConfigurePrecompiles(new(big.Int).SetUint64(parent.Time()), types.NewBlockWithHeader(header), env.state)
+	err = w.chainConfig.ConfigurePrecompiles(new(big.Int).SetUint64(parent.Time()), types.NewBlockWithHeader(header), env.state)
+	if err != nil {
+		werr := fmt.Errorf("failed to configure precompiles: %w", err)
```

Comment:
same comment here since we're using the same wrapping we can just move this into `ConfigurePrecompiles`

Reply:
changed